.github/workflows/lint.yaml

```
@@ -0,0 +1,155 @@
+name: lint
+on: [push, pull_request]
+jobs:
+  check-licences:
+    name: Check Licences
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.10'
+      - name: Enable Problem Matcher for GitHub annotations
+        run: echo "::add-matcher::.github/problem-matcher-lint-check-licences.json"
+      - name: Check licenses
+        shell: bash
+        run: ./scripts/enforce_licence.py
+  generic-nolints:
+    name: Count generic NOLINTs
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Enable Problem Matcher for GitHub annotations
+        run: echo "::add-matcher::.github/problem-matcher-lint-generic-nolints.json"
+      - name: Count the number of generic NOLINTs
+        shell: bash
+        run: ./scripts/count_generic_nolints.sh
+  build:
+    name: Build for Lint Tasks
+    runs-on: ubuntu-latest
+    container: debian:stable
+    steps:
+      - name: Update package database
+        run: apt-get update -y
+      # See comments beginning at https://github.com/actions/runner/issues/763#issuecomment-1435474884
+      # Without Git, actions/checkout@v3 will resort to REST and will not
+      # create a .git folder or .git.config. The Problem Matcher looks for
+      # .git/config to find where the root of the repo is, so it must be
+      # present.
```

If we're going for something like 80 characters we ought to try and hit it more I'd think!
```suggestion
      # See comments beginning at
      # https://github.com/actions/runner/issues/763#issuecomment-1435474884
      # Without Git, actions/checkout@v3 will resort to REST and will not
      # create a .git folder or .git.config. The Problem Matcher looks for
      # .git/config to find where the root of the repo is, so it must be
      # present.
```

---

Fixed!